README.md

```diff
@@ -121,7 +121,16 @@ You may use the flag `--help` to view all supported features.
 
 ### Debugging
 
-The flag `--debug` enables installation of OpenSK in debug mode. You may use the GUI application J-Link RTT viewer to see the debugging messages.
+The flag `--debug` enables installation of OpenSK in debug mode. 
+
+* On nRF52840-DK board: The [Segger RTT debugging protocol](https://www.segger.com/products/debug-probes/j-link/technology/about-real-time-transfer/)
+is used. You may open the GUI application [J-Link RTT viewer](https://www.segger.com/products/debug-probes/j-link/tools/rtt-viewer/) 
+(included in the [J-Link software package](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack))
+to see the debugging messages.
+* On nRF52840-Dongle: The debugging is instead UART based and you will need an
+external hardware (e.g. [Segger J-Link Debug Probe](https://www.segger.com/products/debug-probes/j-link/))
+as mentioned in the detailed installation guide. We do not cover the details of
```

Comment:
Not covering the details is fine. If you have a source to start reading, you could link it. If not, fine with me.

Reply:
I haven't found any proper tutorial or post, just some issues on forums or an informal youtube video https://www.youtube.com/watch?v=No8WCeZaTKo showing the connections.

Reply:
About the dongle part, the J-Link is needed if you want to flash the dongle or to debug the firmware through GDB. The UART is needed to see the debug messages and won't use the J-Link probe but another piece of hardware.